Docs/debugging.md

```diff
@@ -0,0 +1,77 @@
+# Debugging
+
+The content of this guide will help you to configure your PHPStorm in order to debug the PIM.
+The following doc is about Linux only and is not tested on other OS.
```

Comment:
Maybe you could be less reductive: `is tested with Linux, but be adapted for other host systems`.